apps/communications/dialer/test/marionette/lib/views/dialer_options/accessors.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+/* global module */
+
+var SELECTORS = Object.freeze({
+  main: '#views',
+  goToCallLog: '#option-recents',
+  goToContactList: '#option-contacts'
+  
+});
+
+function DialerOptionsAccessor(client) {
+  this.client = client;
+}
+
+DialerOptionsAccessor.prototype = {
```

Comment:
Even though the selector call them actions, I'd rename this class to something like "TabsAccessors". "Tabs" is also something defined in the DOM:
`<section id="views" role="tablist" class="bb-tablist">`

And it's sounds also like a less generic name. What do you think?